topics/synthetic-biology/slides/introduction.html

```diff
@@ -0,0 +1,14 @@
+---
+layout: introduction_slides
```

Comment:
Should this file be deleted?

Reply:
Yes, I'll provide slides later. Thank you.

Reply:
slides would be great! This sounds like a very interesting topic, but currently not a lot of explanation for people who are new to the topic, some intro slides covering the basic concepts involved in synthetic biology would be great to have, and maybe also slides with the tutorials (or a bit more background/explanation in the tutorial itself?)
 
 great work here!

Reply:
Yes I agree. I just added new slides for Synthetic Biology Introduction and more background/explanation in the tutorial. Thanks again.